toolchains/BUILD

```diff
@@ -36,3 +38,79 @@ toolchain(
     toolchain = "@buildbuddy_toolchain//:ubuntu_local_cc_toolchain",
     toolchain_type = "@bazel_tools//tools/cpp:toolchain_type",
 )
+
+toolchain(
+    name = "cosmo_toolchain",
+    toolchain = ":cosmo_cc_toolchain",
+    toolchain_type = "@bazel_tools//tools/cpp:toolchain_type",
+)
```

Comment:
Probably need to specify that this is compatible with linux only? or is everything universal?

Reply:
It should work for everything x86_64. I think it can be configured to be fully universal, but I'm not there yet.